test/RecipeRepository-test.js

```diff
@@ -19,7 +24,7 @@ describe('RecipeRepository', () => {
   });
 
   it('should take in a parameter of recipe data', () => {
-    expect(repository.recipeData).to.equal(sampleData);
+    expect(repository.recipeData).to.equal(recipeList);
```

Comment:
Fixing all the instances where we'd used sampleData to our new array of instances (recipeList)